src/main/java/dev/tricht/lunaris/listeners/CurrencyStackListener.java

```diff
@@ -29,7 +30,9 @@ public void onEvent(GameEvent event) {
             elements.put(new Icon(item, 48), new int[]{0, 0});
             elements.put(new ItemName(item, 48 + Icon.PADDING), new int[]{1, 0});
             elements.put(new Price(item), new int[]{1, elements.size() - 1});
-            elements.put(new Source("poe.ninja"), new int[]{1, elements.size() - 1});
+            if (PropertiesManager.getProperty("trade_search.poe_ninja", "1").equals("1")) {
+                elements.put(new Source("poe.ninja"), new int[]{1, elements.size() - 1});
+            }
```

Comment:
I would leave this in as the currency stack source is really poe.ninja

Do you agree and if so could you remove the if statement here?

Reply:
Yes, it really was unnecessary statement.